.github/workflows/ci-check.yml

```diff
@@ -17,43 +17,15 @@ jobs:
         run: npm run format:check
       - name: Run Linter
         run: npm run lint
-      - name: Type Check
+      - name: Run Type Check
         run: npm run type-check
-      - name: Jest Test
+      - name: Run Unit Tests
         run: npm run test
-  check-lockfile:
```

Comment:
Copied check-lockfile and check-req-json jobs to `ci-build.yml`, so it runs on PRs instead of pulls